src/pyOpenMS/tests/unittests/test_ProteaseDigestion.py

```
@@ -0,0 +1,64 @@
+    ff.setSpecificity(1) # Semi-specific
+    output = []
+    ff.digest(seq, output)
+    # MHAR, LVP, HAR, LV, AR, L, R, MHA, VP, MH, P, M
+    assert len(output) == 12
+
+    assert pyopenms.ProteaseDigestion().peptideCount is not None
+    ff.setSpecificity(2)
+    assert ff.peptideCount(seq) == 2
+
+    assert pyopenms.ProteaseDigestion().isValidProduct is not None
+    assert ff.isValidProduct(seq, 0, 4, true, false, false)
```

Ruff (0.12.2)

Undefined name `true`

(F821)